src/Organ.java

```diff
@@ -18,8 +22,10 @@ public Organ(int health, int capacity, double resistance, int priority, String n
     public int getCurrentCapacity() { return currentCapacity; }
     public int getPriority() { return priority; }
     public double getResistance() { return resistance; }
-    public double getPlayerControl() { return Math.round(((double)currentHealth / maxHealth) * 100.0) / 100.0; }
+    public double getPercentHealth() { return Math.round(((double)currentHealth / maxHealth) * 100.0) / 100.0; }
     public String getName() { return name; }
+    public List<ImmuneCell> getCellList() { return cellList; }
+    public List<Pathogen> getPathogenList() { return pathogenList; }
 
     public boolean changeCapacity(int delta) {
```

Comment:
For the changeHealth method, modify so method changes health by percents as well.